services/insights-handler/main.go

```diff
@@ -0,0 +1,178 @@
+	flag.StringVar(&mqHost, "rabbitmq-hostname", "localhost", "The hostname for the rabbitmq host.")
+	flag.StringVar(&mqPort, "rabbitmq-port", "5672", "The port for the rabbitmq host.")
+	flag.IntVar(&mqWorkers, "rabbitmq-queue-workers", 1, "The number of workers to start with.")
+	flag.IntVar(&rabbitReconnectRetryInterval, "rabbitmq-reconnect-retry-interval", 30, "The retry interval for rabbitmq.")
+	flag.IntVar(&startupConnectionAttempts, "startup-connection-attempts", 10, "The number of startup attempts before exiting.")
+	flag.IntVar(&startupConnectionInterval, "startup-connection-interval-seconds", 30, "The duration between startup attempts.")
+	flag.StringVar(&lagoonAPIHost, "lagoon-api-host", "http://localhost:3000/graphql", "The host for the lagoon api.")
+	flag.StringVar(&jwtTokenSigningKey, "jwt-token-signing-key", "super-secret-string", "The jwt signing token key or secret.")
+	flag.StringVar(&jwtAudience, "jwt-audience", "api.dev", "The jwt audience.")
+	flag.StringVar(&jwtSubject, "jwt-subject", "actions-handler", "The jwt audience.")
+	flag.StringVar(&jwtIssuer, "jwt-issuer", "actions-handler", "The jwt audience.")
+	flag.StringVar(&insightsQueueName, "insights-queue-name", "lagoon-insights:items", "The name of the queue in rabbitmq to use.")
+	flag.StringVar(&insightsExchange, "insights-exchange", "lagoon-insights", "The name of the exchange in rabbitmq to use.")
+	flag.StringVar(&secretAccessKey, "secret-access-key", "minio123", "s3 secret access key to use.")
+	flag.StringVar(&s3Origin, "s3-host", "localhost:9000", "The s3 host/origin to use.")
+	flag.StringVar(&accessKeyID, "access-key-id", "minio", "The name of the bucket to use.")
+	flag.StringVar(&bucket, "s3-bucket", "lagoon-sboms", "The s3 aws region.")
+	flag.StringVar(&region, "s3-region", "", "The s3 region.")
+	flag.BoolVar(&disableS3Upload, "disable-s3-upload", false, "Disable uploading insights data to an s3 bucket")
+	flag.BoolVar(&disableApiIntegration, "disable-api-integration", false, "Disable insights data integration for the Lagoon API")
+	flag.Parse()
+
+	// get overrides from environment variables
+	mqUser = getEnv("RABBITMQ_USERNAME", mqUser)
+	mqPass = getEnv("RABBITMQ_PASSWORD", mqPass)
+	mqHost = getEnv("RABBITMQ_ADDRESS", mqHost)
+	mqPort = getEnv("RABBITMQ_PORT", mqPort)
+	lagoonAPIHost = getEnv("GRAPHQL_ENDPOINT", lagoonAPIHost)
+	jwtTokenSigningKey = getEnv("JWT_SECRET", jwtTokenSigningKey)
+	jwtAudience = getEnv("JWT_AUDIENCE", jwtAudience)
+	jwtSubject = getEnv("JWT_SUBJECT", jwtSubject)
+	jwtIssuer = getEnv("JWT_ISSUER", jwtIssuer)
+	insightsQueueName = getEnv("INSIGHTS_QUEUE_NAME", insightsQueueName)
+	insightsExchange = getEnv("INSIGHTS_EXCHANGE", insightsExchange)
+	secretAccessKey = getEnv("S3_FILES_SECRET_ACCESS_KEY", secretAccessKey)
+	s3Origin = getEnv("S3_FILES_HOST", s3Origin)
+	accessKeyID = getEnv("S3_FILES_ACCESS_KEY_ID", accessKeyID)
+	bucket = getEnv("S3_FILES_BUCKET", bucket)
```

Comment:
If we're using the same bucket as the task and build logs (which we probably should for now), we need to make sure we're creating a subfolder for sboms - e.g. https://github.com/uselagoon/lagoon/blob/main/services/logs2s3/src/readFromRabbitMQ.ts#L47